Gopkg.lock

```diff
@@ -15,46 +15,14 @@
 
 [[projects]]
   name = "github.com/Azure/go-autorest"
-  packages = [
-    "autorest",
-    "autorest/adal",
-    "autorest/azure",
-    "autorest/date",
-    "autorest/to"
-  ]
+  packages = ["autorest","autorest/adal","autorest/azure","autorest/date","autorest/to"]
```

Comment:
This is my first time using `dep` on an upstream project and `dep ensure` is formatting them as such. This is most likely due to me not being familiar enough with the tool to know the formatting options for the tool